### PR TITLE
Implement new design on resources & data page

### PR DESCRIPTION
**Pull request by peetucket**
Screenshot of how it looks in this PR:

Screen Shot 2022-02-28 at 1 02 57 PM

Astrid's design in Slack: https://stanfordlib.slack.com/archives/C024NAC2CP3/p1645681360546819

Also, made some tweaks to make things look better on mobile, most notably, small adjustments to text sizes on the home page, and hiding some images when the screen gets smaller.

e.g.

Screen Shot 2022-02-28 at 10 33 14 PM

Screen Shot 2022-02-28 at 10 33 21 PM

### CODE REVIEW

**Comment by peetucket** [comment]
Removed the Stanford only video and centered the diagram on the page:

Screen Shot 2022-03-01 at 8 06 58 AM

Screen Shot 2022-03-01 at 8 07 14 AM

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -109,6 +117,11 @@ a:hover {
 	color: #000;
 }
 
+.bubbles h2 a {
+	color: #000;
+	font-size: 1.3rem;
+}
+
```
Comment (peetucket) [inline app/assets/stylesheets/application.css:124]: slightly smaller "bubble" font sizing on home page to help with mobile

File: app/views/default/data.html.erb [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
-                <%=link_to image_tag('blm.png', alt: "Black Lives Matter"), "https://guides.library.stanford.edu/SystemicRacismTracker"%>
+        <div class="row gx-4 justify-content-center">
+            <div class="col-6 pb-4">
+              <div class="red-banner-body">
+                <p class="red-banner-title">KNOW Systemic Racism Additional Resources</p>
+                    <ul>
+                        <li class="pb-6 pt-4"><%=link_to "Library Guide", "https://guides.library.stanford.edu/SystemicRacismTracker"%></li>
+                        <li><%=link_to "Related projects happening on campus", "https://news.stanford.edu/2022/02/01/examining-systemic-racism-advancing-racial-equity/"%></li>
+                        <li><%=link_to "Say Their Names Spotlight Exhibit", "https://exhibits.stanford.edu/saytheirnames"%></li>
+                    </ul>
```
Comment (peetucket) [inline app/views/default/data.html.erb:13]: link the headers (as well as the images), and then apply classes to hide the images on narrow width screens

File: app/views/default/index.html.erb [hunk 3]
```diff
@@ -19,19 +19,19 @@
     <div class="container">
         <div class="row">
             <div class="col-3 bubbles rounded text-center">
-                <h2>Learn about systemic racism</h2>
-                <%= link_to image_tag('button_timeline.png', class: "img-fluid pt-5", alt: "timeline"), :timeline %>
+                <h2><%= link_to "Learn about systemic racism", :timeline%></h2>
+                <%= link_to image_tag('button_timeline.png', class: "img-fluid pt-5 d-none d-md-block", alt: "timeline"), :timeline %>
             </div>
```
Comment (peetucket) [inline app/views/default/index.html.erb:24]: link the headers and then apply classes to hide the images on narrow width screens

File: app/views/default/data.html.erb [hunk 4]
```diff
@@ -1,31 +1,46 @@
...
         <div class="row pt-4 justify-content-center">
-            <div class="col-6 text-center">
+            <div class="col-12 text-center">
                 <h3>Examples of policies and how they intersect</h3>
-                <%=image_tag "housing_policing_venn_diagram.png", width: "100%", alt: "Housing/Policing Venn Diagram" %>
-            </div>
-            <div class="col-6 text-center">
-                <h3>Watch a short film on predictive policing and "algorithmic violence"</h3>
-                <iframe src="https://embed.stanford.edu/iframe?url=https://purl.stanford.edu/gq999bh9311&hide_title=true" height="300px" width="80%" title="Media viewer" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" allowfullscreen></iframe>
+                <%=image_tag "housing_policing_venn_diagram.png", width: "60%", alt: "Housing/Policing Venn Diagram" %>
```
Comment (peetucket) [inline app/views/default/data.html.erb:43]: remove stanford only video and center venn diagram

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**